boards/rpi-pico/Makefile.include

```diff
@@ -1,2 +1,7 @@
 CPU_MODEL       := RP2040
 PORT_LINUX      ?= /dev/ttyUSB0
+
+ifeq ($(PROGRAMMER),jlink)
+  JLINK_DEVICE = RP2040_M0_0
+  FLASHFILE = $(HEXFILE)
```

Comment:
Funny, yesterday there was some discussion in matrix and there seemed to be a consensus to prefer ELF files for flashing unless there is a reason against ELF files. Admittedly, only few actively participated in the discussion, though.

But if the ELF file works equally well, I'd personally prefer to stick with the ELF file for flashing.

Reply:
Yup, especially because creating a .hex is an extra step (that takes time, ...)

Reply:
That's because of this discussion that I think about it. But I don't have strong opinions over .elf vs .hex. I wanted to use J-link probe for sometimes now with this board so it ease debugging when working on new peripheral drivers.
I can confirm that it works too with .elf file so I'll update this PR.

Reply:
And squashed directly.